frontend/public/locales/config.ts

```diff
@@ -0,0 +1,56 @@
+import i18next from 'i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import { initReactI18next } from 'react-i18next';
+// import clusterDe from './de/cluster.json';
+// import cluster from './en/cluster.json';
+// import clusterEs from './es/cluster.json';
+
+// https://www.i18next.com/how-to/add-or-load-translations#add-after-init
+
+// export const resources = {
+//   en: {
+//     cluster,
+//   },
+//   de: {
+//     clusterDe,
+//   },
+//   es: {
+//     clusterEs,
+//   },
+// } as const;
```

Comment:
Can we make this somehow more automatic? (maybe listing the folder, or at least automating the `cluster{LANG}` resource imports/uses)